### ISSUE TITLE
Update to the 24.12 Ban List

### ISSUE THREAD

**Issue by lcrandall** [mtgred/netrunner#7906]
There were no un-bans so its not preventing games from happening in standard but it would be nice to have the current ban list.

**Comment by NBKelly** [comment]
See: https://github.com/NoahTheDuke/netrunner-data/pull/60

**Comment by NBKelly** [comment]
Closed as completed (just need to wait on the server reset now :))
